README.md

```diff
@@ -104,16 +104,16 @@ You can use the prebuilt test tool as shown below.
 ./tools/teste2e/bin/teste2e
 ```
 
-This will create 10 hosts with 1 VM each.
+This will create 5 hosts with 10 VM each.
 Once everything is created, you can test network connnectivity as shown below.
 
 1. Run ```kubectl get pods -A``` to see all vhost pods.
 ![Step 1](docs/images/merak_e2e_step1.jpg)
 
-1. Merak uses network namespaces to emulate VMs, run ``` kubectl exec -it vhost-0 ip netns exec v000 ip a ``` to get the IP address of the emulated VM `v000` inside of the emulated host `vhost-0`.
+1. Merak uses network namespaces to emulate VMs, run ``` kubectl exec -it -n (pod namespace) vhost-0 ip netns exec v000 ip a ``` to get the IP address of the emulated VM `v000` inside of the emulated host `vhost-0`.
```

Comment:
@phudtran it better to use `<namespace of the pod>`

Reply:
Updated